index.html

```diff
@@ -352,9 +352,28 @@ <h2 class="section-title">SCHEDULE</h2>
                               </p>
                             </div>
                           </li>
-                          <li class="no-toggle">
-                            <p class="time">11:30 AM</p>
-                            <h3 class="schedule-tbd">Details coming soon ...</h3>
+                          <li class="toggle">
+                            <div class="clearfix">
+                              <div class="col-10 col">
+                                <p class="time">11:30 AM</p>
+                                <h3>Crafting your DEV identity</h3>
+                              </div>
+                              <div class="col-2 col">
+                                <p class="speakers">Jess Lee</p>
+                                <ul class="qcSchSpeakers clearfix">
+                                  <li>
+                                    <a href="#jess-lee" class="tips" title="JESS LEE">
+                                      <img src="./assets/img/speakers/jess-lee.jpg" alt="Jess Lee headshot" nopin="nopin"/>
```

Comment:
I deployed this to staging and Jess Lee's pic is broken. Maybe because the photo is `.JPG` instead of the `.jpg` in the image tab?